### PR TITLE
Dirty fix for Fix KeyError: 'filename' & Add link to the cookie helper plugin

### PR DESCRIPTION
**Pull request by badchoices**
I added a link to help new users get their auth token information using a browser extension developed by @M-rcus

### CODE REVIEW

File: modules/onlyfans.py [hunk 1]
```diff
@@ -865,7 +865,14 @@ def compare_metadata(new_metadata: create_metadata, old_metadata: create_metadat
...
 # Scrapes the API for content
-
+unknowncount = 0
+def getValueIfKey(key, dictv):
+    global unknowncount
+    if key in dictv:
+        return dictv[key]
+    else:
+        unknowncount+=1
+        return 'unknown{unkowncount}'
```
Comment (nappyapp) [inline modules/onlyfans.py:875]: typo? shouldn't it be `{unknowncount}` not `{unkowncount}`?

**Review by nappyapp — CHANGES_REQUESTED** [review]

File: modules/onlyfans.py (hunk 1 above)
Comment (badchoices) [reply modules/onlyfans.py]: You are right. I have fixed that. I did this PR in a rush.

**Comment by sintaxx** [comment]
I'm using the latest version from the releases page, how do i apply this fix to mine ?

**Comment by hirezz** [comment]
Just tried this. I received the error
File ":\OnlyFans-master\modules\onlyfans.py", line 1084, in media_scraper
    if found_medias:
UnboundLocalError: local variable 'found_medias' referenced before assignment

**Comment by DIGITALCRIMINAL** [comment]
Thanks, but it's been resolved now.

**Pull request closed without merging**